scripts/install_foundry.sh

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env bash
+# Copyright (C) 2023, Ava Labs, Inc. All rights reserved.
+# See the file LICENSE for licensing terms.
+
+set -e
+
+FOUNDRY_VERSION=v0.1.0
```

Comment:
should we set this in `versions.sh`?

Reply:
IMO we should only use `versions.sh` for a version that is needed in more than one place. Adding indirectly here only increases complexity.